src/interfaces/IRoot.sol

```diff
@@ -0,0 +1,90 @@
+// SPDX-License-Identifier: AGPL-3.0-only
```

Comment:
This file can be removed right?

Reply:
Actually `IERC7540Vault` uses `IRecoverable` from this file. We're not using `IERC7540Vault` in these examples though. Is it okay to remove it from IERC7540.sol? Or should I redeclare `IRecoverable` in IERC7540.sol?

Reply:
Ah nvm I see you recommend removing IERC7540Vault below.